Tests/Data/HydroMechanics/Verification/hm1_3Dgravity.prj

```diff
@@ -275,71 +340,13 @@
     <parameters>
         <parameter>
             <name>E</name>
-            <type>Group</type>
-            <group_id_property>MaterialIDs</group_id_property>
-            <index_values>
-                <index>0</index>
-                <value>1.0e+10</value>
-            </index_values>
-            <index_values>
-                <index>1</index>
-                <value>1.0e+10</value>
-            </index_values>
-            <index_values>
-                <index>2</index>
-                <value>1.0e+10</value>
-            </index_values>
-            <index_values>
-                <index>3</index>
-                <value>1.0e+10</value>
-            </index_values>
+            <type>Constant</type>
+            <value>1e10</value>
         </parameter>
         <parameter>
             <name>nu</name>
-            <type>Group</type>
-            <group_id_property>MaterialIDs</group_id_property>
-            <index_values>
-                <index>0</index>
-                <value>0.25</value>
-            </index_values>
-            <index_values>
-                <index>1</index>
-                <value>0.25</value>
-            </index_values>
-            <index_values>
-                <index>2</index>
-                <value>0.25</value>
-            </index_values>
-            <index_values>
-                <index>3</index>
-                <value>0.25</value>
-            </index_values>
-        </parameter>
-        <parameter>
-            <name>k</name>
-            <type>Group</type>
```

Comment:
I'd keep the Group type parameter. There is no need to repeat all other properties four times, just because one parameter is heterogeneous.

Reply:
But either way I have to mention in every medium either the constant value or the reference to the parameter, so the medium section will be large anyway. Then I thought, it would be best to choose the constant value, to get rid of the large parameter section.